### PR TITLE
added Numbers filter rule

### PR DESCRIPTION
**Pull request by skoop**
Adding a new rule that returns all numbers from a given string

### CODE REVIEW

File: docs/filter-rules.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ filters, take a look at the callback filter-rule, or check out "Extending the Fi
 * [trim](#trim)
 * [upper](#upper)
 * [upperFirst](#upperfirst)
+* [numbers](#numbers)
```
Comment (rick-nu) [inline docs/filter-rules.md]: Can you please keep this list in alphabetic order? :innocent: thanks!

Comment (skoop) [inline docs/filter-rules.md]: done

File: tests/FilterRule/NumbersTest.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        $this->filter = new Filter();
+    }
+
+    /**
+     * @dataProvider getNumbersResults
+     * @param string $value
+     * @param string $excludeTags
+     * @param string $filteredValue
+     */
+    public function testStripHtmlFilterRule($value, $filteredValue)
```
Comment (rick-nu) [inline tests/FilterRule/NumbersTest.php]: `testNumbersFilterRule`

Comment (skoop) [inline tests/FilterRule/NumbersTest.php]: oops, fixed

**Comment by rick-nu** [comment]
Very nice, thank you!

**Pull request merged**